packages/common/test/mocks/agolItems.ts

```diff
@@ -708,7 +708,7 @@ export function getAGOLGroup(): any {
     owner: "LocalGovTryItLive",
     description: "Description of an AGOL group",
     snippet: "Snippet of an AGOL group",
-    typeKeywords: ["JavaScript"],
```

Comment:
Why move "JavaScript" from typeKeyword to tag?  For WMAs, e.g., the typeKeyword is set behind the scenes to "JavaScript" whether or not the user sets it also as a tag.

Reply:
@MikeTschudi that function gets a group. Group doesn't have typeKeywords only tags.

Reply:
Thanks!